Shared/Samples/Update related features/UpdateRelatedFeaturesView.swift

```diff
@@ -0,0 +1,296 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct UpdateRelatedFeaturesView: View {
+    /// The model that holds the data for displaying and updating the view.
+    @State private var model = Model()
+    
+    /// A Boolean value indicating whether the feature data is being loaded.
+    @State private var isLoading = false
+    
+    /// The error shown in the error alert.
+    @State private var error: (any Error)?
+    
+    /// The last locations in the screen and map where a tap occurred.
+    @State private var lastSingleTap: (screenPoint: CGPoint, mapPoint: Point)?
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(map: model.map)
+                .onSingleTapGesture { screenPoint, mapPoint in
+                    lastSingleTap = (screenPoint, mapPoint)
+                }
+                .callout(placement: $model.calloutPlacement) { _ in
+                    // Show a callout with editable content when a feature is selected.
+                    if model.calloutIsVisible {
+                        calloutContent
+                    }
```

Comment:
```suggestion
                    // Show a callout with editable content when a feature is selected.
                    calloutContent
```
Checking `calloutIsVisible` is redundant because the callout only shows when the callout is not `nil`. Please remove the `calloutIsVisible` property from the model.